modules/@apostrophecms/doc-type/index.js

```diff
@@ -784,6 +786,10 @@ module.exports = {
 
         await self.apos.schema.convert(req, schema, input, doc);
 
+        if (options.createId) {
+          doc.aposDocId = options.createId;
+        }
```

Comment:
We can now specify an `aposDocId` when creating a document. It allows us to create a doc with the ID of the original. We don't want to localize since it's not a copy of an existing doc.